# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../audacious/cleanup.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- audacious-1.0.0/Plugins/Effect/audiocompress/audacious-glue.c_orig	2006-04-18 20:02:18.000000000 +0200
+++ audacious-1.0.0/Plugins/Effect/audiocompress/audacious-glue.c	2006-04-20 17:50:00.000000000 +0200
@@ -151,7 +151,8 @@
 
 void freePrefs(CompressorPrefs * prefs)
 {
-	gtk_widget_destroy(prefs->dialog);
+	if(inited)
+		gtk_widget_destroy(prefs->dialog);
 
 }
 
